app/views/pages/webhooks.html.md

```diff
@@ -0,0 +1,109 @@
+
+When a notification is received at the supplier's endpoint, it must:
+
+1. Verify the PECS-SIGNATURE header of the notification
+2. Upon verification return an HTTP success code (in the range 200-299)
+
+The notification is not considered to have been delivered until an HTTP success is received.
+
+**_The endpoint should return success (or failure) immediately after signature verification and success should not be contingent on subsequent processing._** 
+
+I.e. if a notification was successfully received and the signature verified, it is not correct to return a failure code because of some other problem in the subsequent processing as this would compromise the decoupled nature of the systems.
+
+An example notification message is given below:
+
+    POST / HTTP/1.1
+    Host: foobar.requestcatcher.com
+    Accept: */*
+    Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+    Connection: keep-alive
+    Content-Length: 415
+    Content-Type: application/vnd.api+json
+    Keep-Alive: 30
+    Pecs-Notification-Id: 2cb108dd-8d47-4a5f-8d36-29324a770f05
+    Pecs-Signature: 5cWQEe9emC7Myvj8jxVDIWI0jxoshOhitXfsCQBtTS4=
+    User-Agent: pecs-webhooks/v1
+    {"data":{"id":"2cb108dd-8d47-4a5f-8d36-29324a770f05","type":"notifications","attributes":{"event_type":"create_move","timestamp":"2020-02-18T11:05:00+00:00"},"relationships":{"move":{"data":{"id":"149f1c27-1b7d-4c60-a4d4-ae8afbe92501","type":"moves"},"links":{"self":"http://hmpps-book-secure-move-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk/api/v1/moves/149f1c27-1b7d-4c60-a4d4-ae8afbe92501"}}}}}
+
+
+## 4. Verification of the PECS-SIGNATURE header
+It is necessary to verify the signature of the notification with the following algorithm in order to guarantee the authenticity of the message:
+
+1. Calculate the SHA-256 HMAC of the message body using the pre-agreed <SECRET> when the subscription was created
+2. Base64 encode the calculated HMAC
+3. Check that the encoded value matches the PECS-SIGNATURE header. If it does match, return an HTTP success status (e.g. 202 - Accepted). If it does not match, return an error code (e.g. 403 - Forbidden).
+
+Ruby code for calculating the signature is given below:
+
+    require 'base64'
+    require 'openssl'
+    expected_signature = Base64.strict_encode64(OpenSSL::HMAC.digest('sha256', secret, body))
+
+
+## 5. Idempotent nature of notifications
+The supplier should store the `id` (also in the `PECS-NOTIFICATION-ID` header) of every successful notification received. If a subsequent notification with the same `id` is received, it should be ignored.
```

Comment:
Should we be clear that a `200` response should be returned for ignored notifications?